src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -647,6 +666,43 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
             }
         }
 
+#if !UNIX
+        private bool WebBrowserAvailable()
+        {
+            RegistryKey regKey = null;
+            try
+            {
+                // Query the registry and remove the quotes '"' from the path.
+                regKey = Registry.ClassesRoot.OpenSubKey("HTTP\\shell\\open\\command", false);
+                string browserPath = regKey.GetValue(null).ToString().Replace((char)34 + "", "");
+                if (browserPath != null)
+                {
+                    if (!browserPath.EndsWith(".exe", StringComparison.OrdinalIgnoreCase))
+                    {
+                        // Remove any extra chars in the path after ".exe".
+                        browserPath = browserPath.Substring(0, browserPath.LastIndexOf(".exe") + 4);
```

Comment:
4 is a magic number here.
It's better to re-write it as

```
const string exeExtention = ".exe";
browserPath = browserPath.Substring(0, browserPath.LastIndexOf(exeExtension) + exeExtension.Length);
```
Same goes about the "34".
It' good that there is a comment above, but if it's a comment, it should be very close to the number, i.e.

```
.Repalce(/* " */ (char)34 + ...)
```